### PR TITLE
[W6.1e][W14-B1] Ngo Phuong Thao

### PR DESCRIPTION
**Pull request by Sisyphus25**
- Add JUnit test for Utils#isAnyNull(Object...) method

### CODE REVIEW

**Comment by kowshik-sundararajan** [comment]
Kindly update the title of the pull request to follow Phase B format.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +36,36 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHasAnyNull() throws Exception {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:40]: The test itself is correctly written,
But, is `elementsHasAnyNull` the name of the method you are testing?
- Remember that the first part of the name is the name of the method you are testing.
- Please also be aware of the 3 part test name format that may be useful when you write tests for your project.

**Review by okkhoy — COMMENTED** [review]
Please update the title of the PR to reflect your phase B team name.

**Comment by okkhoy** [comment]
@Sisyphus25 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**